Python/initconfig.c

```diff
@@ -646,7 +648,7 @@ config_init_defaults(PyConfig *config)
 
     config->isolated = 0;
     config->use_environment = 1;
-    config->use_peg = 0;
+    config->use_peg = 1;
```

Comment:
We can probably delete this entirely now, right?

Reply:
Yup! Thanks for the catch, is getting very late and I am starting to miss more things that I would like :(

Reply:
It is indeed getting late. Already after 4 over here.